doc/api/http.md

```diff
@@ -747,6 +747,10 @@ added: v0.1.90
 Begin accepting connections on the specified `port` and `hostname`. If the
 `hostname` is omitted, the server will accept connections on any IPv6 address
 (`::`) when IPv6 is available, or any IPv4 address (`0.0.0.0`) otherwise.
+
+*Note*: in most operating systems, listening to any IPv6 address (`'::'`) may
```

Comment:
I think using `any IPv6 address` is just for historical reason: C socket() use the const  `INADDR_ANY`.

Actually i want to change all `any IPv? address` to 
`the IPv? address INADDR_ANY(::)`, but i can not, just because this term are referenced in several places.